frontend/public/components/app.jsx

```diff
@@ -179,6 +180,9 @@ class App extends React.PureComponent {
             <Route path="/k8s/all-namespaces/customresourcedefinitions/:plural" exact component={ResourceListPage} />
             <Route path="/k8s/all-namespaces/customresourcedefinitions/:plural/:name" component={ResourceDetailsPage} />
 
+            <Route path="/k8s/ns/:ns/vmtemplates" exact component={VirtualMachineTemplatesPage} />
```

Comment:
Does it need to be registered at `/k8s/ns/:ns/vmtemplates` ? The plural is just `templates` so I don't think there can be conflict with anything else from OKD.
If I don't miss something, we can stay consistent and use `/k8s/ns/:ns/templates`. Means, these 2 lines can be removed as would be handled by a generic rule bellow.

Reply:
We are talking about subset of templates https://github.com/kubevirt/web-ui/pull/114/files#diff-215e6ee0ab95eeb805c45846a7521fe6R69 and i would like to avoid possible future conflicts if tectonic ever adds generic templates page. Also IMHO it makes sense to have vmtemplates not just templates in URL as it right away makes clear that this page is about VM templates, not all templates